External/Plugins/CodeRefactor/Controls/MoveDialog.cs

```diff
@@ -121,7 +121,15 @@ void FillTree()
                 });
             }
             if (classpaths.Count > 0) tree.Items.AddRange(classpaths.ToArray());
-            if (tree.Items.Count > 0) tree.SelectedIndex = 0;
+            if (tree.Items.Count > 0)
+            {
+                tree.SelectedIndex = 0;
+                this.processButton.Enabled = true;
+            }
+            else
+            {
+                this.processButton.Enabled = false;
```

Comment:
Remove this. to be consistent with the rest of the file. Also, it seems to me this could be rewritten to:

```
if (classpaths.Count > 0)
{
    tree.Items.AddRange(classpaths.ToArray());
    tree.SelectedIndex = 0;
    this.processButton.Enabled = true;
}
else this.processButton.Enabled = false;
```
I don't think it's possible for `classpaths` to have at least 1 item and then tree be empty heh.